R/pkg/R/generics.R

```diff
@@ -771,6 +771,10 @@ setGeneric("over", function(x, window) { standardGeneric("over") })
 
 ###################### WindowSpec Methods ##########################
 
+#' @rdname orderBy
+#' @export
```

Comment:
I checked the generated docs, and it showed in both docs, but I do agree that it is not necessary considering the purpose of `setGeneric`. Then we have to decide where to put it...